airbyte-cdk/python/README.md

```diff
@@ -49,9 +49,14 @@ See the [concepts docs](docs/concepts/) for a tour through what the API offers.
 
 ### First time setup
 
```

Comment:
why are these required? it seems really undesirable to have system level deps like this one.Any way we can avoid them?

Reply:
This is an unfortunate dependency for https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/extractors/jq.py#L30

I'll look into [jello](https://github.com/kellyjonbrazil/jello) as an alternative as it only runs in the python interpreter

Reply:
did a [quick spike](https://github.com/airbytehq/airbyte/commit/043b3434f922213320cbc72d9bd65fa0fa43b7d3) and I think jello is a good python-only tool. I'll have a more in-depth replacement proposal after hackdays

Reply:
Whats the effort to move to `jello`

Reply:
@bazarnov @koconder the need for the extra step was removed as part of https://github.com/airbytehq/airbyte/pull/13570